### DISCUSSION TITLE
Probleme avec le SQL

### DISCUSSION THREAD

**Discussion by KevinJ-Dev** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#311] [Q&A]
Bonjour/Bonsoir

J'ai ressèment pris le base SOZ afin de participer à l'amélioration de SOZ, mais je rencontre un soucie avec la BDD/SQL, j'ai donc suivie les étapes d'installation, et j'ai des erreur comme ceci dans le terminal de mon hébergeur et au moment d'exécuter des commandes telle que dans le soz-core dans le terminal de mon environnement de développement.

yarn run prisma migrate deploy
yarn run prisma db seed

Error sur vscode

![4](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/079767e5-f999-4cde-8e5b-58efd01ef838)

Error hébergeur

![1](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/6fb2b35c-da83-4374-b67b-426ac49e711e)
![2](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/225e59c7-7725-4ba8-a62b-97429d657daa)
![3](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/32522780/59cf9d3b-8343-4251-89c3-e3f0b275740f)

**Comment by Laikker** [comment]
Tu as bien créé un fichier .env à partir du .env-dist dans le répertoire soz-core?

**Reply by KevinJ-Dev** [reply]
Oui dans le quel il y a seulement la DATABASE_URL

**Reply by Laikker** [reply]
et elle commence bien par mysql://?

**Reply by KevinJ-Dev** [reply]
Oui il commence par mysql://
Je viens de refaire un yarn install que j'ai bien delete avant le node_module et le .json
Je viens de voir que dans le package.json est vide, j'ai fait un yarn build par tout hasard, et j'ai ceci
Usage Error: Couldn't find a script named "build" (ce qui est normal le json est vide)

**Reply by Laikker** [reply]
tu as fais yarn install?

**Reply by KevinJ-Dev** [reply]
Oui et j'ai bien de nouveau le node module avec le json

**Reply by Laikker** [reply]
Je comprends pas comment tu fais pour avoir un package.json vide

**Reply by KevinJ-Dev** [reply]
Je t'avoue moi non plus c'est la première fois que j'ai ça, je viens de re transvasé juste le soz-core par défaut sur la machine, et je te redis ça, quand j'aurais fait de nouveau les manipulation à faire (ça va prendre un peu de temps :/ )

**Reply by KevinJ-Dev** [reply]
Résolu merci à Laikker d'avoir porter son aide :)
